doc/getting-started/apl-nuc.rst

```diff
@@ -450,11 +450,6 @@ partition. Follow these steps:
          You should run ``dmesg |grep ttyS0`` to get port address from the output, and then
          add the ``uart`` parameter into the ``efibootmgr`` command.
 
-   #. ``vuart=ttySn@irqN``: this tells the hypervisor which virtual serial device SOS
```

Comment:
Line 432 above mentions three options (and with this change there are only two)

Reply:
Fixed, thanks.